### PR TITLE
fix code format (exclude pfto)

### PR DESCRIPTION
**Pull request by linshokaku**
Changed pysen black and isort to enable.

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -5,12 +5,16 @@ requires = ["setuptools<64", "wheel"]
...
 enable_mypy = true
 mypy_preset = "strict"
 line_length = 80
 
 [[tool.pysen.lint.mypy_targets]]
   paths = ["pytorch_pfn_extras"]
+
+[tool.pysen.lint.source]
+  includes = ["."]
+  excludes = ["pytorch_pfn_extras/onnx/", "tests/pytorch_pfn_extras_tests/onnx_tests/"]
```
Comment (linshokaku) [inline pyproject.toml:20]: Where to change lint settings

File: .github/workflows/pretest-and-test.yml [hunk 2]
```diff
@@ -38,7 +38,7 @@ jobs:
 
     - name: Code Style
       run: |
-        pip install pysen black==21.11b1 flake8==4.0.1 isort==5.10.1 mypy==0.991
+        pip install pysen black==23.3.0 flake8==4.0.1 isort==5.10.1 mypy==0.991
```
Comment (linshokaku) [inline .github/workflows/pretest-and-test.yml:41]: Change the version of black to eliminate problems caused by dependencies with click.

**Review by emcastillo — DISMISSED** [review]

**Comment by emcastillo** [comment]
sorry, can we solve the conflictS? I will run CI and merge after that!

**Comment by linshokaku** [comment]
Conflicts resolved. Please review again!

**Review by emcastillo — APPROVED** [review]

**Pull request merged**